release-info/howto.md

```diff
@@ -406,9 +406,9 @@ See the email announce templates in the `templates/` directory.
 
 See
 
-- templates/beta.md for alpha and beta releases
-- templates/rc.md for release candidate
-- templates/production.md for the production release
+- [templates/beta.md](templates/beta.md) for alpha and beta releases
+- [templates/rc.md](templates/rc.md) for release candidate
+- [templates/production.md](templates/production.md) for the production release
```

Comment:
I think this may work with just `[templates/beta.md]`, but I haven't checked.

Reply:
It doesn't work